source/class/qx/tool/cli/commands/package/Install.js

```diff
@@ -282,7 +282,7 @@ qx.Class.define("qx.tool.cli.commands.package.Install", {
       if (!release_data) {
         throw new qx.tool.utils.Utils.UserError(`'${repo_name}' has no release '${tag_name}'.`);
       }
-      // TODO: the path in the cache data should be the path to the library containing Manifest.json, not to the Manifest.json itself
+      // TO DO: the path in the cache data should be the path to the library containing Manifest.json, not to the Manifest.json itself
```

Comment:
Should stay TODO so that it can be found by tools

Reply:
I wanted to remove the linter warnings since the todo is not really important. The whole stuff needs to be rewritten one day, and then the todos will be meaningless.